auto-complete.js

```diff
@@ -62,6 +62,44 @@
             });
         }
 
+
+    window.changeText = () => {
+            
+        const newText = 
+             `
```

Comment:
Do not use ES2015+ syntax features, like arrow function, `const` or template string. Doing this will break the whole JS script for older browsers, all code from the Topbar JS will not work.

Use ES5 syntax.